Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml

```
@@ -667,7 +669,7 @@
       coefficients:
         Blunt: 0.7
         Slash: 0.7
-        Piercing: 0.4
+        Piercing: 0.6
```

```suggestion
        Piercing: 0.6 # Goobstation - From 0.4
```
*fluoride stares at you